packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlFacade.php

```diff
@@ -218,4 +218,15 @@ public function getAbsoluteUrlByFriendlyUrl(FriendlyUrl $friendlyUrl): string
 
         return $domainConfig->getUrl() . '/' . $friendlyUrl->getSlug();
     }
+
+    /**
+     * @param string $routeName
+     * @param int $entityId
+     */
+    public function removeFriendlyUrlsForAllDomains(string $routeName, int $entityId): void
+    {
+        foreach ($this->getAllByRouteNameAndEntityId($routeName, $entityId) as $friendlyUrl) {
+            $this->em->remove($friendlyUrl);
```

Comment:
Probably we should not remove those URLs from database. If you will be changing content on one URL then it can make SEO worse